src/hooks/resources/application.ts

```diff
@@ -53,21 +50,21 @@ sbvrUtils.addPureHook('POST', 'resin', 'application', {
 			throw new Error('App name may only contain [a-zA-Z0-9_-].');
 		}
 
-		return Promise.all([
-			resolveDeviceType(api, request, 'is_for__device_type'),
-			checkDependentApplication(args),
-		])
-			.then(() => {
-				request.values.should_track_latest_release = true;
-				if (request.values.slug == null) {
-					request.values.slug = appName.toLowerCase();
-				}
-			})
-			.tapCatch(err => {
-				if (!(err instanceof ConflictError)) {
-					captureException(err, 'Error in application postparse hook', { req });
-				}
-			});
+		try {
+			await Promise.all([
```

Comment:
Native Promise is currently masked by Bluebird, make sure to remove (or rename) the import.